src/components/TopicSection/TopicSection.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { act, render, screen } from "@testing-library/react";
+import App from "../../App";
+
+describe("Test topic buttons", () => {
+  test("should render topic buttons", async () => {
+    render(<App />);
```

Comment:
Try rendering <TopicSection /> instead of <App />, because we currently don't need App context

Reply:
Yes, I could do that, of course.
As we agreed, I decided to render the whole App, because in the next task I will have communication with the backend app, and in that case it is necessary.